### PR TITLE
[175] Add suite summary page to draws

### LINKED ISSUE #175: Admins should be able to view a summary of the suites in a draw
Child of #167. Admin should be able to:

1. Visit draw page
2. Click link "Add/Edit Suites"
3. View a giant form:
   1. Suites with checkboxes
   2. Suites organized by suite size
   3. Only available suites displayed (or check-able, I guess they could be disabled)
   4. Within suite size, suites organized by whether or not they belong to another draw already
   5. If they belong to a draw, the draw name should be listed
   6. Admins should be able to check/uncheck all within each section (suite size / belonging to draw or not)
4. Click a submit button
5. Profit!

### PR DESCRIPTION
**Pull request by orenyk**
Resolves #175
- Add suite summary page with a listing of all of the suites belonging
  to the draw, separated by suite size
- Add SuitesBySize query object

### CODE REVIEW

File: app/controllers/draws_controller.rb [hunk 1]
```diff
@@ -52,6 +52,11 @@ def filter_intent_report
     render action: 'intent_report'
   end
 
+  def suite_summary
+    @all_sizes = SuiteSizesQuery.call
+    @suites_by_size = SuitesBySizeQuery.new(@draw.suites).call
```
Comment (esoterik) [inline app/controllers/draws_controller.rb]: also, I know we talked about this yesterday, but I still don't see how this excludes suites with groups

**Comment by esoterik** [comment]
rebased!

File: app/controllers/draws_controller.rb (hunk 1 above)
Comment (orenyk) [inline app/controllers/draws_controller.rb]: You're right, I must have done it elsewhere. Updated!

**Comment by orenyk** [comment]
I had to remove the ordering from the `available` scope since it conflicted with the `SuiteSizesQuery` (Postgres errors), so there were some extra adjustments made there. Squashed and ready for re-review (I'll rebase other issues as well).

**Pull request merged**